src/lib/felaExpandCssShorthandsPlugin.ts

```diff
@@ -0,0 +1,47 @@
+import * as _ from 'lodash'
+import * as expand from 'css-shorthand-expand'
+
+export default () => {
+  const expandCssShorthands = styles => {
+    const processedStyles = {}
+
+    Object.keys(styles).forEach(cssPropertyName => {
+      const cssPropertyValue = styles[cssPropertyName]
+
+      if (typeof cssPropertyValue === 'object') {
```

Comment:
to be sure, could you, please, suggest what is the case for it? are we targeting pseudo element styles with this check?